Makefile

```diff
@@ -130,6 +130,9 @@ test-unit: tests/testreport/testreport
 	mkdir -p $$tmp/root $$tmp/runroot; \
 	$(GO) test -v -tags "$(STORAGETAGS) $(SECURITYTAGS)" ./cmd/buildah -args -root $$tmp/root -runroot $$tmp/runroot -storage-driver vfs -signature-policy $(shell pwd)/tests/policy.json -registries-conf $(shell pwd)/tests/registries.conf
 
+vendor-in-container:
+	podman run --privileged --rm --env HOME=/root -v `pwd`:/src -w /src golang make vendor
```

Comment:
Currently, we're using the latest tag which is likely to break us in the future again when go 1.14+ introduces yet another change. It also seems that the Ubuntu package isn't installing registries.conf (at least that's what the error message in the Travis indicates). So instead of `golang`, we could use `docker.io/library/golang:1.13`.

Ultimately, I would prefer using Cirrus for these checks and remove the vendor checks from Travis altogether. This will prevent us from running those checks redundantly on all jobs. In Cirrus, we can specify a job running in a `golang:1.13` container, so there would be no need to install podman.